RazorGenerator.Testing/WebViewPageExtensions.cs

```diff
@@ -158,21 +167,41 @@ private static void EnsureDummyViewEngineRegistered()
             }
         }
         
-        class DummyViewEngine : IViewEngine
+        public class DummyViewEngine : IViewEngine
         {
+            public bool ReturnDummyView = true;
+            public Func<string, IView> ViewFactory = s => null;
```

Comment:
Might be useful to pass in the view type too here. Consider making this the first arg a context type instead.